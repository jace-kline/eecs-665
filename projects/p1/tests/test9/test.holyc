'a 'b 'c '\n '\t '  'x 'y 'z