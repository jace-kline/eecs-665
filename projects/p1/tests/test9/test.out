CHARLIT:a
CHARLIT:b
CHARLIT:c
CHARLIT:newline
CHARLIT:tab
CHARLIT: 
CHARLIT:x
CHARLIT:y
CHARLIT:z
EOF [1,29]
